NSSServices/Controllers/RegressionRegionsController.cs

```diff
@@ -115,8 +114,14 @@ public async Task<IActionResult> Get(int id, [FromQuery] bool getpolygon = false
             try
             {
                 if(id<0) return new BadRequestResult(); // This returns HTTP 404
+                RegressionRegion entity = agent.GetRegressionRegion(id, getpolygon).FirstOrDefault();
```

Comment:
I'm wondering what your thoughts are on changing the getpolygon =false query parameter to something that isn't "action" oriented, but rather something like includegeometry=false changing it from 'polygon' to geometry, also makes it a bit more vague

Reply:
I agree - I'll get that changed.